pkg/kubelet/kubelet.go

```diff
@@ -1943,3 +1947,222 @@ func (kl *Kubelet) GetRootInfo(req *cadvisorApi.ContainerInfoRequest) (*cadvisor
 func (kl *Kubelet) GetMachineInfo() (*cadvisorApi.MachineInfo, error) {
 	return kl.cadvisor.MachineInfo()
 }
+
+func (kl *Kubelet) SyncRocketPods(allPods []api.BoundPod, podSyncTypes map[types.UID]metrics.SyncPodType, start time.Time) error {
```

Comment:
@sym3tri  It should be.  I should `s/SyncPods/SyncDockerPods` and `s/SyncRocketPods/SyncPod`.

But I am just want doing `SyncRocketPods` to highlight the new code path :D